.hashibot.hcl

```
@@ -818,6 +821,11 @@ behavior "pull_request_path_labeler" "service_labels" {
       "**/*_codestar_*",
       "**/codestar_*"
     ]
+    "service/codestarconnections" = [
+      "aws/internal/service/codestarconnections_/**/*",
```

Trailing underscore:

```suggestion
      "aws/internal/service/codestarconnections/**/*",
```